src/polybench/POLYBENCH_2MM.cpp

```diff
@@ -79,7 +79,11 @@ POLYBENCH_2MM::POLYBENCH_2MM(const RunParams& params)
 
 POLYBENCH_2MM::~POLYBENCH_2MM()
 {
+}
 
+Index_type POLYBENCH_2MM::getProblemSize() const
```

Comment:
So all the polybench kernels return zero in this PR? Is the idea that I come in after this is merged, adjust the polybench default kernel sizes to be ~1M for each and then return the correct value from getProblemSize()?

Reply:
Ya whichever PR comes in second can fix up these values, I didn't want to redo any work you were doing with the polybench kernels.

Reply:
I'm going to wait until your stuff is merged because I don't want to deal with all the merge conflicts.